src/gui/utils.cpp

```diff
@@ -159,15 +160,22 @@ void Utils::Gui::openFolderSelect(const Path &path)
     }
 
 #ifdef Q_OS_WIN
-    HRESULT hresult = ::CoInitializeEx(nullptr, COINIT_MULTITHREADED);
-    PIDLIST_ABSOLUTE pidl = ::ILCreateFromPathW(reinterpret_cast<PCTSTR>(path.toString().utf16()));
-    if (pidl)
+    auto thread = std::unique_ptr<QThread>(QThread::create([path]()
```

Comment:
`std::make_unique()`?

Reply:
> `std::make_unique()`?

It will not work in the case of a static method is used to create object.